test/stage2/cbe.zig

```diff
@@ -824,6 +824,28 @@ pub fn addCases(ctx: *TestContext) !void {
         , "");
     }
 
+    {
+        var case = ctx.exeFromCompiledC("@ptrToInt", .{});
+
+        case.addCompareOutput(
+            \\pub export fn main() i32 {
+            \\    var y = 0x41;
+            \\    _ = @ptrToInt(&y);
+            \\    return 0;
+            \\}
+        , "");
+
+        case.addError(
```

Comment:
We don't really want to add tests for TODOs, but nice thought :)

Reply:
I removed this test case, and added TODO in the error message.